app/src/main/java/bubbleheads/buy_cook/RecipeDetailedFragment.java

```diff
@@ -0,0 +1,30 @@
+package bubbleheads.buy_cook;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+
+public final class RecipeDetailedFragment extends Fragment {
+    private Recipe detailedRecipe;
+
+    @Override
+    public View onCreateView(final LayoutInflater inflater,
+                             final ViewGroup container,
+                             final Bundle savedInstanceState) {
+        final View view = inflater.inflate(R.layout.detailed_recipe_fragment, container, false);
+        detailedRecipe = ((MainActivity) getActivity()).getChosenRecipe();
+        getActivity().setTitle(detailedRecipe.getRecipeName());
+        final ImageView detailedRecipeImage = (ImageView) view.findViewById(R.id.detailed_recipe_image);
+        detailedRecipeImage.setImageResource(detailedRecipe.getRecipePhoto());
+        final TextView recipeDetail = (TextView) view.findViewById(R.id.detailed_recipe_detail);
+        recipeDetail.setText(detailedRecipe.getRecipeDetail());
```

Comment:
recipeDetail  and detailed Recipe  : it would be good to rename the first variable